mypy-nofollow.ini

```diff
@@ -21,7 +21,7 @@ files =
 
 # Minimum version supported - variable annotations were introduced
 # in Python 3.8
-python_version = 3.8
```

Comment:
We still need to support 3.8. You should be able to handle everything with typing_extension backports and not using the latest and flashiest typing syntax.